projects/m1/005-units-of-time-again/python/main.py

```diff
@@ -0,0 +1,16 @@
+#ask to user how many seconds in order to perform computatio
+seconds = int(input("Please insert seconds:\n"))
+seconds_day = 86400
+seconds_hour = 3600
+seconds_minute = 60
```

Comment:
Only for information, consider that when a **constant** has a **fixed value, known beforehand** (e.g. Pi, seconds per minute, days in a year, etc.), it can be written in **uppercase letters**. However, it is acceptable to use **lowercase** letters when the constant is assigned a **customs value**.

Reply:
ok :) thanks